test/acceptance/tests.py

```diff
@@ -1102,6 +1105,43 @@ def test_train_self_peer_staff(self, peer_grades_me):
             ])
 
 
+@ddt.ddt
+class MultipleOpenAssessmentTest(OpenAssessmentTest, FullWorkflowMixin):
+    """
+    Test the multiple peer-assessment flow.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentTest, self).setUp('multiple_ora')
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    def set_vert_no(self, vert_no):
+        """ Set vertical number """
+        self.submission_page.vert_no = vert_no
+        self.self_asmnt_page.vert_no = vert_no
+        self.peer_asmnt_page.vert_no = vert_no
+        self.student_training_page.vert_no = vert_no
+        self.staff_asmnt_page.vert_no = vert_no
+        self.grade_page.vert_no = vert_no
+        self.staff_area_page.vert_no = vert_no
+
+
+    @retry()
+    @attr('acceptance')
+    @ddt.data(True, False)
+    def test_multiple_ora_complete_flow(self, peer_grades_me):
+        """
+        Scenario: complete workflow on a unit containing multiple ORA blocks.
+        """
+        self.set_vert_no(0)
+        # First Vertical - which is selected by default
+        learner = self.do_train_self_peer(peer_grades_me)
+
+        # Second Vertical
+        self.set_vert_no(1)
+        self.do_train_self_peer(peer_grades_me)
```

Comment:
@mushtaqak This test is only doing `train_self_peer`, but the problem you've added has a staff step enabled as well. It seems odd to only test a problem partway through.

Can you add functionality to this test to staff assess the user as well, and ensure:
- After the first assessment is complete, a grade appears for it but not for the second assessment
- After the second assessment is complete, grades appear for both
This will be similar to our full workflow tests: https://github.com/edx/edx-ora2/blob/master/test/acceptance/tests.py#L1078-L1102